schema/deploy/tables/revision_status.sql

```diff
@@ -0,0 +1,23 @@
+-- Deploy cif:tables/revision_status to pg
+
+begin;
+
+alter table cif.amendment_status rename to revision_status;
+alter table cif.revision_status add column is_amendment_specific boolean;
+
+update cif.revision_status set is_amendment_specific = false where name='Draft' or name='Applied';
+update cif.revision_status set is_amendment_specific = true where name not in('Draft', 'Applied');
```

Comment:
We should pick a coding style, I like the `in ( ... , ... )`, it reads better and is more concise

```suggestion
update cif.revision_status set is_amendment_specific = false where name in ('Draft', 'Applied');
update cif.revision_status set is_amendment_specific = true where name not in ('Draft', 'Applied');
```